proto/src/determined/run/v1/run.proto

```diff
@@ -110,4 +111,6 @@ message FlatRun {
   bool parent_archived = 18;
   // Data related the the experiment associated with this run.
   optional FlatRunExperiment experiment = 19;
+  // The archived status of this run
```

Comment:
nit: add that this is only looking at the archived status at the run level and not taking into account whether the experiment is archived or not.

Reply:
fixed